package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "storefrontapp",
-  "version": "0.0.0",
+  "version": "4.1.0",
```

Comment:
```suggestion
  "version": "0.0.0",
```
AFAIK we aren't updating this package for some reason

Reply:
thanks Michal, will update it.